src/app/header/search/search.component.html

```diff
@@ -5,4 +5,5 @@
     <option value="">Food</option>
```

Comment:
Create ENUM or constatn file with all values and viewValues

Reply:
will do, when will be doing this component